src/test/rpc_tests.cpp

```diff
@@ -308,4 +343,205 @@ BOOST_AUTO_TEST_CASE(rpc_ban)
     BOOST_CHECK_EQUAL(adr.get_str(), "2001:4d48:ac57:400:cacf:e9ff:fe1d:9c63/128");
 }
 
+
+BOOST_AUTO_TEST_CASE(rpc_getblockchaininfo)
+{
+    // test bip109 counting
+
+    // need the chain params for setting up test chains and cases
+    SelectParams(CBaseChainParams::MAIN);
+    const Consensus::Params& params = Params().GetConsensus();
+    // we need 2 more than the size of the majority window because
```

Comment:
you are correct, that comment is obsolete and based on earlier misunderstanding of mine